src/index.jsx

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Header from './header/Header';
+import Footer from './components/footer/Footer';
+import Header from './components/header/Header';
+import MovieList from './components/movie-list/MovieList';
+import ErrorBoundary from './components/error-component/ErrorBoundary';
+import './index.css'
 
 ReactDOM.render(
   <React.StrictMode>
-    <Header/>
+  
+  <Header/>
+  <ErrorBoundary>
+    <MovieList/>
+    </ErrorBoundary>
+    <Footer/>
```

Comment:
Indent the code in these lines.
Also, those react components that are self-closing needs to have a space before closing or ending tag like
<Header /> instead of <Header/>